fluster/decoders/gstreamer.py

```diff
@@ -120,14 +120,19 @@ def parse_videocodectestsink_md5sum(self, data: List[str], verbose: bool) -> str
                 "conformance/checksum, checksum-type=(string)MD5, checksum=(string)"
             )
             sum_start = line.find(pattern)
-            if sum_start > 0:
-                sum_start += len(pattern)
-                sum_end = line[sum_start:].find(";")
-                if sum_end > 0:
-                    sum_end += sum_start
-                    md5sum = line[sum_start:sum_end]
-                    if not verbose:
-                        return md5sum
+            if sum_start <= 0:
+                # Skip to the next iteration if sum_start is less than or equal to 0
+                continue
+            sum_start += len(pattern)
+            sum_end = line[sum_start:].find(";")
+            if sum_end <= 0:
+                # Skip to the next iteration if sum_end is less than or equal to 0
+                continue
+            sum_end += sum_start
+            md5sum = line[sum_start:sum_end]
+            if not verbose:
+                print(md5sum)
```

Comment:
I understand that this linear way of coding makes it simple to read, but there is a minus point that i want to avoid.
Without the else statements the cohesion of the code is not apparent.

Imagine that someone removes lines 123-125 accidentally or without checking further logic, this could break the function without us even noticing (python compiler will not complain for syntax)

Change request: add else statements even if we have to have 2 extra nested levels

Let me know if i missed something or you disagree